igor-web/src/main/java/com/netflix/spinnaker/igor/scm/ScmMaster.java

```diff
@@ -0,0 +1,14 @@
+package com.netflix.spinnaker.igor.scm;
+
+import java.util.List;
+
+/** Abstracts underlying implementation details of each SCM system under a common interface. */
+public interface ScmMaster {
+  public static final String DEFAULT_GIT_REF = "refs/heads/master";
+
+  public abstract List<String> listDirectory(
```

Comment:
I might be mistaken here, but if this is an interface. why this methods are abstracts?

Reply:
That was the IntelliJ converter thingie. Thanks for catching it -- will fix!